frontend/packages/operator-lifecycle-manager/src/components/create-operand.tsx

```diff
@@ -581,17 +863,20 @@ export const CreateOperandForm: React.FC<CreateOperandFormProps> = ({
       return (
         <RadioGroup
           currentValue={formValues[field.path]}
-          items={_.values(ImagePullPolicy).map((policy) => ({ value: policy, title: policy }))}
+          items={_.values(ImagePullPolicy).map((policy) => ({
+            value: policy,
+            title: policy,
+          }))}
           onChange={({ currentTarget }) => updateFormValues(field.path, currentTarget.value)}
         />
       );
     }
     if (field.capabilities.includes(SpecCapability.updateStrategy)) {
       return (
         <ConfigureUpdateStrategy
-          strategyType={_.get(formValues, `${field.path}.type`)}
-          maxUnavailable={_.get(formValues, `${field.path}.rollingUpdate.maxUnavailable`)}
-          maxSurge={_.get(formValues, `${field.path}.rollingUpdate.maxSurge`)}
+          strategyType={_.get(formValues, [field.path, 'type'])}
+          maxUnavailable={_.get(formValues, [field.path, 'rollingUpdate', 'maxUnavailable'])}
+          maxSurge={_.get(formValues, [field.path, 'rollingUpdate', 'maxSurge'])}
           onChangeStrategyType={(type) => updateFormValues([field.path, 'type'], type)}
```

Comment:
This is the widget I am having issues with. For some reason, when `updateFormValues` is called, it is receiving the correct arguments, the 'setFormValue' callback is returning the correct updated state, but the update never makes it into the 'formValues' state.